hilo.rb

```diff
@@ -1,24 +1,35 @@
 #!/usr/bin/env ruby
 puts "Welcome to the Hi-Lo game."
 puts "Guess a random number from 1 to 100 (inclusive)."
-puts "You have 6 guesses."
+puts "You will have a total of 6 guesses."
+puts ""
 
 random_num = rand(1..100)
-puts "Please guess a number."
 
-# only allow integer input
-guess = gets
-while guess.strip != guess.to_i.to_s  
-    puts "Guess a number!"
+# gives the user 6 guesses
+for i in 0..5
+    guesses_left = 6 - i
+    puts "You have #{guesses_left} guesses left."
+    puts "Please guess a number."
+
+    # only allow integer input
     guess = gets
-end
-guess = guess.to_i
+    while guess.strip != guess.to_i.to_s  
+        puts "Guess a number!"
```

Comment:
Do we consider decimals not as numbers? Would you consider changing the message to only provide whole number or a number between 1-100.

Reply:
Done in https://github.com/amykingOW/Hi-Lo-Game/pull/3/commits/371f0685d9a4c81bde0ddd626de908efe229e620. The message specifies the user should enter a whole number.